src/components/favorites/dto/list-favorite.dto.ts

```diff
@@ -0,0 +1,40 @@
+import { Field, InputType, ObjectType } from '@nestjs/graphql';
+import { Type } from 'class-transformer';
+import { ValidateNested } from 'class-validator';
+import {
+  PaginatedList,
+  SecuredList,
+  SortablePaginationInput,
+} from '../../../common';
+import { Favorite } from './favorite';
+
+@InputType()
+export abstract class FavoriteFilters {
```

Comment:
I can see no use case for this search filter

Reply:
I would search for filters belonging to one User.  Then I might filter it by type of BaseNode.  For Example.  Favorite Locations, Favorite Projects, Favorite Languages.  
These options might be an enum, that is not every existing type.  
The ones I would suggest would be 

- Project
Language
Organization
Location
Film
Story
LiteracyMaterial
User

All that to say, If we require this, It should be a new ticket, not this ticket.  

I would remove the baseNodeId as a filter